airbyte-integrations/connectors/source-marketo/source_marketo/source.py

```diff
@@ -336,14 +346,10 @@ def get_json_schema(self) -> Mapping[str, Any]:
             "activityDate": {"type": ["null", "string"], "format": "date-time"},
             "activityTypeId": {"type": ["null", "integer"]},
             "campaignId": {"type": ["null", "integer"]},
-            "attributes": {"type": ["null", "string"]},
```

Comment:
Why deleted attributes?

Reply:
as I understand we are removing `attributes` from record object

https://github.com/airbytehq/airbyte/blob/65ef9295d16691b0889492c9f6331bc3c1668dbb/airbyte-integrations/connectors/source-marketo/source_marketo/source.py#L223

we need only flattened fields from `attributes`
but `attributes` itself can be removed

maybe I missed something ?